react-components/src/library/components/materials-collection/sortable-materials-collection-list-row.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { SortableElement } from 'react-sortable-hoc'
+import css from './style.scss'
+
+class UnortableMaterialsCollectionListRow extends React.Component {
```

Comment:
```suggestion
class UnsortableMaterialsCollectionListRow extends React.Component {
```

However the name Unsortable here confuses me. It isn't that it isn't possible to sort this row, it is just that it hasn't been wrapped with the SortableElement, right?  
Would it work if we just called it `MaterialsCollectionListRow`, or if that is confusing how about something like `BaseMaterialsCollectionListRow`?